internal/utils/misc.go

```diff
@@ -304,7 +304,7 @@ func InstallOrUpgradeDeno(ctx context.Context, fsys afero.Fs) error {
 
 	if _, err := fsys.Stat(denoPath); err == nil {
 		// Upgrade Deno.
-		cmd := exec.CommandContext(ctx, denoPath, "upgrade", "--version", "1.27.1")
+		cmd := exec.CommandContext(ctx, denoPath, "upgrade", "--version", "1.28.0")
```

Comment:
Should we just remove the `--version` flag? That way it stays recent (i.e. always on stable)

Reply:
we'd probably want to keep it the same version as `functions serve` (set via Docker image). I think it's better to set the version.